packages/types/src/pricing/service.ts

```diff
@@ -1274,6 +1274,53 @@ export interface IPricingModuleService {
    */
   deleteMoneyAmounts(ids: string[], sharedContext?: Context): Promise<void>
 
+  /**
+   * This method soft deletes money amounts by their IDs.
+   *
+   * @param {string[]} ids - The IDs of the money amounts to delete.
+   * @param {Context} sharedContext - A context used to share resources, such as transaction manager, between the application and the module.
+   * @returns {Promise<void>} Resolves when the money amounts are successfully deleted.
+   *
+   * @example
+   * import {
+   *   initialize as initializePricingModule,
+   * } from "@medusajs/pricing"
```

Comment:
comment: At some point we will have to start using the medusa app like:
```ts
const { modules } = await MedusaApp({
    modulesConfig,
    servicesConfig: joinerConfig,
    injectedDependencies,
  })
```

not now but just to say